pkg/retester/retester.go

```diff
@@ -202,6 +204,13 @@ func NewController(ghClient githubClient, cfg config.Getter, gitClient git.Clien
 	return ret
 }
 
+func newBackoffCache(awsSession *session.Session, cacheFile string, cacheRecordAge time.Duration, logger *logrus.Entry) backoffCache {
```

Comment:
NIT: we do not need a new function here. Moving it into the caller is easier to read the code.